modules/terraform/output.go

```diff
@@ -131,6 +100,32 @@ func parseMap(m map[string]interface{}) (map[string]interface{}, error) {
 	return result, nil
 }
 
+func parseList(items []interface{}) (_ []interface{}, err error) {
+	for i, v := range items {
+		rv := reflect.ValueOf(v)
+		switch rv.Kind() {
```

Comment:
Missing case for `default` :thinking: 

should be value copied as is?
```
default:
  result[i] = v
```

Reply:
no we don't need the default case, since we are not copying or moving the array item, but instead overriding the item(s) if it match certain conditions. so if doesn't match the conditions, we don't do anything and the value is intact. Similar to this https://go.dev/play/p/D6cnrJMiltP